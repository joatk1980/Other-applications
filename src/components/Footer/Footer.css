.footer {
  background-color: purple;
  padding: 10px 20px;
  font-size: calc(10px + 20min);
  color: white;
}
