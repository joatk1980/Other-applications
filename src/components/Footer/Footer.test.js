import renderer from "react-test-renderer";
import Footer from "./Footer";

it("Renders the footer", () => {
  const component = renderer.create(<Footer />);
  expect(component).toMatchSnapshot();
});
