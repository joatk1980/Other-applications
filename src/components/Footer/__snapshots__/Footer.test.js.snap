// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the footer 1`] = `
<footer
  className="footer"
>
  <p>
     Â© Weather App. All Rights Reserved 2024.
  </p>
</footer>
`;
